Formula/x/xgo.rb

```diff
@@ -0,0 +1,37 @@
+class Xgo < Formula
+  desc "AI-native programming language that integrates software engineering"
+  homepage "https://xgo.dev/"
+  url "https://github.com/goplus/xgo/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "e7d80a6760f794ffb4f30c89097b8903f8a1e8c6b60706673d88bcdc304f05c8"
+  license "Apache-2.0"
+  head "https://github.com/goplus/xgo.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
```

Comment:
I figure this might be inherited, but looks like the default livecheck just works fine.
```suggestion
```